src/networking.c

```diff
@@ -2768,7 +2769,6 @@ int processInputBuffer(client *c) {
 
         /* Multibulk processing could see a <= 0 length. */
         if (c->argc == 0) {
-            freeClientArgvInternal(c, 0);
```

Comment:
@sundb  i think `freeClientArgvInternal` is unnecessary here, since we actually don't have parsed argv, so we actually do nothing in `freeClientArgvInternal`

Reply:
you're right, we can delete it.

Reply:
thank you